fv3gfs-physics/setup.py

```diff
@@ -30,7 +30,8 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
     ],
-    description="fv3gfs-physics is a gt4py-based physical parameterization for atmospheric models",
+    description="fv3gfs-physics is a gt4py-based physical parameterization \
+        for atmospheric models",
```

Comment:
Nit: You're not wrong but this is the style I see us use most often.
```suggestion
    description=(
        "fv3gfs-physics is a gt4py-based physical parameterization "
        "for atmospheric models"
    )
```